packages/preprocess/src/svelte-ts-preprocess.ts

```diff
@@ -142,7 +142,7 @@ export function readConfigFile(env: Env, path?: string) {
 export interface PreprocessOptions {
   compilerOptions: ts.CompilerOptions
   env: Env
-  showErrors: boolean
+  hideErrors: boolean
```

Comment:
Can you make `hideErrors` as optional property?
```typescript
export interface PreprocessOptions {
  compilerOptions: ts.CompilerOptions
  env: Env
  hideErrors?: boolean
}
```

Reply:
It shouldn't be optional there.
The ops object is already `Partial<PreprocessOptions>`

Reply:
You are right, sorry

Reply:
np :)